### PR TITLE
UI-7471 - Bump @activeviam dependencies

### PR DESCRIPTION
**Pull request by mtadams007**
In order to migrate filters, I had to bump up the version of 5.1. ~I took the opportunity to also upgrade 5.0 to 5.0.19.~ A test failed when bumping up 5.0.15 to 5.0.19. That's out of scope for what I wanted to accomplish here so I reverted that change.

### CODE REVIEW

File: src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInMdx.ts [hunk 1]
```diff
@@ -32,8 +34,8 @@ export const migrateCalculatedMeasuresInMdx = (
       removeCalculatedMemberDefinition(acc, {
         dimensionName: "Measures",
         hierarchyName: "Measures",
-        namePath: [calculatedMeasureName],
-        cube: getCube(dataModel, cubeName),
+        calculatedMemberName: calculatedMeasureName,
+        cube: getCube(getIndexedDataModel(dataModel), cubeName),
```
Comment (antoinetissier) [inline src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInMdx.ts:38]: Can you add a TODO to pass the indexed `dataModel` to the function directly ?
We'll wire it when we're finished with the refactoring how to generically migrate from 5.x to 5.y

File: src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInWidgets.ts [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
   deserializeWidgetState,
   isWidgetWithQueryState,
   MdxSelect,
   WidgetWithQueryState,
 } from "@activeviam/activeui-sdk-5.0";
+import { DataModel } from "@activeviam/activeui-sdk-5.1";
 import { migrateCalculatedMeasuresInMdx } from "./migrateCalculatedMeasuresInMdx";
 import { produce } from "immer";
 import _mapValues from "lodash/mapValues";
-import { serializeWidgetState } from "@activeviam/activeui-sdk-5.1";
```
Comment (RaspberryChaos) [inline src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInWidgets.ts:13]: @antoinetissier I thought we decided [here](https://github.com/activeviam/activeui-migration/pull/49#discussion_r1029280504) in #49  to call `deserializeWidgetState` from 5.0 and `serializeWidgetState` from 5.1, what is the reason for this change ?

File: src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInDashboards.test.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import { uiDashboardsFolder } from "../__test_resources__/uiDashboardsFolder";
 import { migrateCalculatedMeasuresInDashboards } from "./migrateCalculatedMeasuresInDashboards";
-import { dataModelsForTests } from "@activeviam/data-model-5.0";
+import { sandboxDataModel } from "@activeviam/data-model-5.1/dist/__test_resources__";
```
Comment (mtadams007) [inline src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInDashboards.test.ts:3]: Do we need to get it from `dist/__test_resources__`?

File: src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInDashboards.ts [hunk 4]
```diff
@@ -2,12 +2,12 @@ import {
   ContentRecord,
   CubeName,
   DashboardState,
-  DataModel,
+  serializeDashboardState,
   deserializeDashboardState,
   MdxSelect,
   WidgetWithQueryState,
 } from "@activeviam/activeui-sdk-5.0";
-import { serializeDashboardState } from "@activeviam/activeui-sdk-5.1";
```
Comment (RaspberryChaos) [inline src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInDashboards.ts:10]: Same question here, we decided [here](https://github.com/activeviam/activeui-migration/pull/52#discussion_r1039853461) to use `deserializeDashboardState` from 5.0 and `serializeDashboardState` from 5.1, what is the reason for this change ?

File: src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInDashboards.ts [hunk 5]
```diff
@@ -74,7 +77,8 @@ export const migrateCalculatedMeasuresInDashboards = (
         dashboardPages[pageId].content = updatedWidgets;
       }
       dashboardsContent[dashboardId].entry.content = JSON.stringify(
-        serializeDashboardState(deserializedDashboardState),
+        // Casting as a DashboardState from 5.1 as this function is concerned with only updating calculated measures.
```
[the remark this answers is not in the document]
Comment (RaspberryChaos) [reply src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInDashboards.ts]: Ok, I see why this is necessary 👍

File: src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInDashboards.ts [hunk 6]
```diff
@@ -2,12 +2,12 @@ import {
...
   CubeName,
   DashboardState,
-  DataModel,
+  serializeDashboardState,
   deserializeDashboardState,
   MdxSelect,
   WidgetWithQueryState,
 } from "@activeviam/activeui-sdk-5.0";
-import { serializeDashboardState } from "@activeviam/activeui-sdk-5.1";
+import { DataModel } from "@activeviam/activeui-sdk-5.1";
```
[the remark this answers is not in the document]
Comment (antoinetissier) [reply src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInDashboards.ts:10]: The `dataModel` is indexed in `5.1` and NOT indexed in `5.0`.
We want to use the indexed version here.

File: src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInDashboards.test.ts (hunk 3 above)
Comment (antoinetissier) [reply src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInDashboards.test.ts:3]: It's the standard that we started doing recently in the monorepo, to avoid having it end up in the application bundle.

Comment (mtadams007) [reply src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInDashboards.test.ts:3]: Sweet. Looks good to me in that case. I can't approve as I'm the author, but I'll leave a checkmark here. :heavy_check_mark:

**Review by antoinetissier — APPROVED** [review]
Looks good to me in this state ✅

File: yarn.lock [hunk 7]
```diff
@@ -4124,6 +4084,17 @@
     classnames "^2.2.6"
     rc-util "^5.9.4"
 
+"@ant-design/icons@^4.7.0":
```
Comment (RaspberryChaos) [inline yarn.lock:4087]: Is there a reason why we're adding different versions of the same dependency ?
